docs/docs/getting-started/overview/host-architecture.md

```diff
@@ -78,80 +74,32 @@ type host struct {
 }
 ```
 
-#### Status Submodule
-
-The **status submodule** implements the protocol for the status message exchange.
-
-```go=
-type status struct {
-	host          *host
-	hostMessage   *StatusMessage
-	peerConfirmed *sync.Map // map of peer.ID to time.Time
-	peerMessage   *sync.Map // map of peer.ID to *StatusMessage
-}
-```
-
-#### Gossip Submodule
-
-The **gossip submodule** stores information about past messages to effectively determine which messages to propagate and which to ignore.
-
-```go=
-type gossip struct {
-	host    *host
-	hasSeen *sync.Map
-}
-```
-
-#### Syncer Submodule
-
-The **syncer submodule** tracks requested blocks and coordinates syncing with the **core service**.
-
-```go=
-type syncer struct {
-	host              *host
-	blockState        BlockState
-	requestedBlockIDs *sync.Map       // track requested block id messages
-	syncChan          chan<- *big.Int // chain synchronization channel
-}
-```
 
 ### RPC Service
 
-The **rpc service** is an implementation of the RPC server.
-
-- the **rpc service** only reads from block state
-- the **rpc service** only reads from storage state
-- the **rpc service** only reads from network state
+The **rpc service** is an implementation of the <a target="_blank" rel="noopener noreferrer" href="https://github.com/w3f/PSPs/blob/master/PSPs/drafts/psp-6.md">JSON-RPC PSP</a>.
 
 ### State Service
 
 The **state service** is the source of truth for all chain and node state.
 
-- only the **core service** writes to block state
-- only the **core service** writes to storage state
-- only the **network service** writes to network state
-
-## Node Channels
 
-### Block Channels
+## Block production 
 
-_work in progress_
+<img src="/assets/img/block_production.png" alt="block production" />
```

Comment:
This image doesn't load.

Reply:
Unable to replicate
![image](https://user-images.githubusercontent.com/10461622/113168483-fe7d7b80-9244-11eb-9d1c-a0a9a272db46.png)